components/org.wso2.carbon.identity.captcha/src/main/java/org/wso2/carbon/identity/captcha/internal/CaptchaDataHolder.java

```diff
@@ -37,6 +37,8 @@ public class CaptchaDataHolder {
 
     private boolean reCaptchaEnabled;
 
+    private boolean reCaptchaEnterpriseEnabled;
```

Comment:
reCaptchaEnabled and reCaptchaEnterpriseEnabled are two different options. So, the name should imply that.